script/playercomponent.js

```diff
@@ -126,10 +127,31 @@ define(
       function setCurrentTime (time) {
         userInteracted = true;
         if (transitions().canBeginSeek()) {
-          playbackStrategy.setCurrentTime(time);
+          if (windowType !== WindowTypes.STATIC && getLiveSupport(device) === LiveSupport.RESTARTABLE) {
```

Comment:
Per my above comment in `dynamicwindowutils.js`, this should only apply for native strategy, not TAL.